api/v1/requests/AttendeeRequest.js

```diff
@@ -2,28 +2,24 @@ const Request = require('./Request');
 const Attendee = require('../models/Attendee');
 const validators = require('../utils/validators');
 
-const extraInfoValidations = {
+const longFormValidations = {
```

Comment:
Aren't these the validations for the extraInfo? long forms are significantly longer than 255 chars

Reply:
Changed to 1023 to match length in database. 9e5061570fe18f00fd04b5bf446e701b8072b3c4